common/include/pcl/common/file_io.h

```diff
@@ -39,11 +39,10 @@
 #pragma once
 
 #include <iostream>
-#ifndef _WIN32
-  #include <dirent.h>
-#endif
 #include <vector>
 #include <algorithm>
+#include <boost/filesystem.hpp>
```

Comment:
Not needed here but in the impl file since that's where boost is used.

Reply:
Done